staging/src/k8s.io/apiserver/pkg/server/filters/timeout.go

```diff
@@ -143,13 +143,13 @@ func (t *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}()
 
 		postTimeoutFn()
-		tw.timeout(err)
```

Comment:
cherry-pick conflict: in 4.11+ this is a little different. PTAL at the other patchsets and let me know if my resolution here looks appropriate.